examples/qml_features/rust/src/singleton.rs

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+// ANCHOR: book_macro_code
+#[cxx_qt::bridge(cxx_file_stem = "rust_singleton")]
+pub mod ffi {
+    // TODO: automatically inject this when singleton is specified?
```

Comment:
yes I think that would be a better solution

Reply:
Done